osu.Game.Tests/Visual/SongSelect/TestScenePlaySongSelect.cs

```diff
@@ -588,6 +589,65 @@ public void TestAutoplayViaCtrlEnter()
             AddAssert("mod disabled", () => songSelect.Mods.Value.Count == 0);
         }
 
+        [Test]
+        public void TestCinemaViaCtrlShiftEnter()
+        {
+            addRulesetImportStep(0);
+
+            createSongSelect();
+
+            AddStep("press ctrl+shit+enter", () =>
```

Comment:
that's *quite* a typo

```suggestion
            AddStep("press ctrl+shift+enter", () =>
```